### PR TITLE
Add 2 games to the README

### PR DESCRIPTION
**Pull request by tommybuonomo**
Add Wormo and Ripple Effect Puzzle games to the Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,10 +128,21 @@
 
 - Dino Run! - [Android](https://play.google.com/store/apps/details?id=io.github.epsi95.dinorun), - You are already familiar with this game, you can play it in Chrome when there is no internet. I just added a little color to it. Also you can shout to make the little cute Dino to jump, isn't that cool?. By [Probhakar Sarkar](https://github.com/epsi95)
 
+- Wormo [Android](https://play.google.com/store/apps/details?id=com.tbuonomo.wormo), [iOS](https://apps.apple.com/us/app/wormo-the-game/id1530797595) - Meet Wormo the gooey worm in this wonderful casual game ! Think you can put wormo in the nest? Think again, it requires an extraordinary technique. 
+By Tommy Buonomo ([Frenchie Games])
+
+#### Reflexion
```
Comment (spydon) [inline README.md:134]: Reflection?
Puzzle games is maybe a better name for the category?

**Review by spydon — APPROVED** [review]
Looks good, and congratulations to the releases again!

File: README.md [hunk 2]
```diff
@@ -128,10 +128,21 @@
 
 - Dino Run! - [Android](https://play.google.com/store/apps/details?id=io.github.epsi95.dinorun), - You are already familiar with this game, you can play it in Chrome when there is no internet. I just added a little color to it. Also you can shout to make the little cute Dino to jump, isn't that cool?. By [Probhakar Sarkar](https://github.com/epsi95)
 
+- Wormo [Android](https://play.google.com/store/apps/details?id=com.tbuonomo.wormo), [iOS](https://apps.apple.com/us/app/wormo-the-game/id1530797595) - Meet Wormo the gooey worm in this wonderful casual game ! Think you can put wormo in the nest? Think again, it requires an extraordinary technique.
```
Comment (spydon) [inline README.md:131]: Just saw one tiny thing more, there shouldn't me a space after "game".

File: README.md [hunk 3]
```diff
@@ -128,10 +128,21 @@
...
 ## Non-English Content
 
 - [Mandarin Chinese (中文)](https://github.com/flame-engine/awesome-flame/blob/main/CONTENT_ZH.md)
 
 ## Contributing
 
 [Please read the contribution guidelines](https://github.com/flame-engine/awesome-flame/blob/main/CONTRIBUTING.md)
+
+
+[frenchie games]: https://frenchiegames.app/#/
```
Comment (erickzanardo) [inline README.md:148]: This link should be on your entries, and not here, right @impulse ?

Comment (impulse) [reply README.md:148]: Yes

**Comment by spydon** [comment]
@impulse do you intend to update this PR?

**Comment by spydon** [comment]
Sorry tagged the wrong person, @tommybuonomo is the correct one of course.
> do you intend to update this PR?

File: README.md (hunk 3 above)
Comment (spydon) [reply README.md:148]: @tommybuonomo this link still has to be moved.

**Pull request merged**